### DISCUSSION TITLE
is it possible to manipulate the cache files?

### DISCUSSION THREAD

**Discussion by chchia** [qarmin/czkawka#553] [General]
for example, i have a fast local disk, and a slow remote network disk.

this is what i do before i upload the picture to slow remote disk:

1. new picture are stored in the fast local disk.
2. i set slow remote disk as reference, and compare with the local disk, remove any duplicate of similar images.
3. i then upload the local disk to remote disk for long term storage.

problem: i need to hash again all new uploaded picture from slow remote disk, which consume a lot of time.

is there a way to edit/manipulate the cache files? so that i can change the hashed picture from local directory to remote directory without the need of rehash.

![image](https://user-images.githubusercontent.com/1291771/148005901-21d36da9-973f-40d9-96f5-d77ac42e031c.png)
above is the cache file opened in file editor, i can see the picture URI location. can i simply change replace the directory name?

media is slow remote disk
upload is fast local disk

will changing 
home/chchia/upload/2020.05/100/(25).jpg

to
home/chchia/media/2020.05/100/(25).jpg

works?

**Comment by qarmin** [comment]
Until 3.3.1 version, I used my own implementation which was saved as plain text which was easy to change
Now due bugs I was forced to use binutils crate which by default save files with binary format.
It may be possible to use json format(which takes more space and also more time to load) which would allow to modify results.

**Reply by chchia** [reply]
great, hope to see some other format that may allow user to manipulate directly.

thank you.

**Comment by qarmin** [comment]
Since - https://github.com/qarmin/czkawka/commit/aaa5885326c7d49ec91f8afff6a1ae7aa16d2962 by default results from `bin` file are loaded.
When it is missing, results from json files are loaded(if exists).
To export json files, it is required to enable saving json files in GUI

**Reply by EvanEdwards** [reply]
Is there a way to run the commandline czkawka tool via ssh and then export or otherwise point the gui at the resulting cache so that the GUI can be run on a mounted network drive and not pull over... say... 200TB of data?
